platform-sdk/docs/proposals/migratePlatformStateToProtobuf.md

```diff
@@ -0,0 +1,49 @@
+# Requirements
+
+We need all merkle leaves in the state to be serialized in protobuf format. Currently `PlatformState.java` uses
+the old platform serialization format.
+
+# Proposal
+
+## Protobuf
+
+Migrate the platform state to the protobuf format detailed in this PR: https://github.com/hashgraph/hedera-protobufs/pull/349
+
+This protobuf only contains non-deprecated fields. Some of the fields in the current platform state are no longer in
+use, and these will not be carried forward.
+
+Rosters are notably absent from this protobuf. More on that in the [Rosters](#rosters) section.
+
+## Rosters
+
+When we eventually have TSS, there will be the need for a special purpose map that holds rosters. Currently, we need
```

Comment:
What is the data format for Roster?  How compatible is it, or does it need to be, with config.txt?  What are the format changes that can be incorporated from DAB Phase 2?